src/page-elements/search-page.tsx

```diff
@@ -118,32 +126,69 @@ export default class SearchPage extends React.PureComponent<unknown, State> {
 		});
 	};
 
+	private _onShowExperimental = (): void => {
+		this.setState({ betaResults: !this.state.betaResults });
```

Comment:
This isn't guaranteed to work. Use an updater function instead.
```suggestion
		this.setState(state => ({ betaResults: !state.betaResults }));
```